Update policy-csp-experience.md in regards to Chat icon policy (#10108)
MicrosoftDocs/windows-itpro-docs: windows/client-management/mdm/policy-csp-experience.md

-   2 - Hide: The Chat icon will be hidden by default. Users can show or hide it in Settings.
-   3 - Disabled: The Chat icon will not be displayed, and users cannot show or hide it in Settings.

> Note: Options 1 and 2 (Show, Hide) only work on first boot whereas option 3 - Disabled will work on all attempts.

Reviewer: ```suggestion
> [!NOTE]
> Options 1 and 2 (Show, Hide) only work on the first sign-in attempt whereas Option 3 (Disabled) works on all attempts.
```

-   2 - Hide: The Chat icon will be hidden by default. Users can show or hide it in Settings.
-   3 - Disabled: The Chat icon will not be displayed, and users cannot show or hide it in Settings.

> [!NOTE]

> > Options 1 and 2 (Show, Hide) only work on the first sign-in attempt whereas Option 3 (Disabled) works on all attempts.

Author: ```suggestion
> Option 1 (Show) and Option 2 (Hide) only work on the first sign-in attempt. Option 3 (Disabled) works on all attempts.
```
